package.json

```diff
@@ -27,12 +27,12 @@
     "lint": "eslint --fix --ext .ts .",
     "lint:ci": "eslint -f checkstyle -o reports/checkstyle.xml --ext .ts .",
     "latest:develop-maintenance": "npm install --save-exact @linkurious/rest-client@lk-develop-maintenance",
-    "latest:develop-stable": "npm install --save-exact @linkurious/rest-client@lk-develop-stable",
-    "latest:develop": "npm install --save-exact @linkurious/rest-client@lk-develop",
-    "latest:develop-next": "npm install --save-exact @linkurious/rest-client@lk-develop-next",
+    "latest:develop-stable": "npm install --save-exact @linkurious/rest-client@lk-develop-stable @linkurious/shared@lk-develop-stable",
+    "latest:develop": "npm install --save-exact @linkurious/rest-client@lk-develop @linkurious/shared@lk-develop",
+    "latest:develop-next": "npm install --save-exact @linkurious/rest-client@lk-develop-next @linkurious/shared@lk-develop-next",
     "latest:master-maintenance": "npm install --save-exact @linkurious/rest-client@lk-master-maintenance",
-    "latest:master-stable": "npm install --save-exact @linkurious/rest-client@lk-master-stable",
-    "latest:master": "npm install --save-exact  @linkurious/rest-client@lk-master",
+    "latest:master-stable": "npm install --save-exact @linkurious/rest-client@lk-master-stable @linkurious/shared@lk-master-stable",
+    "latest:master": "npm install --save-exact  @linkurious/rest-client@lk-master @linkurious/shared@lk-master",
```

Comment:
@enys fyi `@linkurious/shared` has been added as a dependency of `@linkurious/ogma-linkurious-parser` with this PR.

I've updated the `latest:*` npm scripts accordingly. I did not change the maintenance scripts has this feature is not targetting maintenance. This is probably going to bite us at some point when we promote 3.2 to stable. Feel free to open a PR with the necessary adjustments (if any) and ping me for review.